reposerver/askpass/server.go

```diff
@@ -88,3 +91,14 @@ func (s *server) getCreds(id string) (*Creds, bool) {
 	creds, ok := s.creds[id]
 	return &creds, ok
 }
+
+// Environ returns the environment variables that should be set when invoking git.
+func (s *server) Environ(id string) []string {
+	return []string{
+		fmt.Sprintf("GIT_ASKPASS=%s", "argocd"),
```

Comment:
nit: any real reason to have this `Sprintf` if there is no variable being used?

Reply:
Good catch, fixed!